index.bs

```diff
@@ -2486,8 +2587,7 @@ a [=trigger state=] |triggerState|:
 
 1. Let |specEntry| be the [=map/entry=] for
     |source|'s [=attribution source/trigger specs=][|triggerState|'s [=trigger state/trigger data=]].
-1. Let |triggerTime| be the greatest [=moment=] that is strictly less than
-    |triggerState|'s [=trigger state/report window=]'s [=report window/end=].
+1. Let |triggerTime| be |triggerState|'s [=trigger state/report window=]'s [=report window/start=].
```

Comment:
Could you clarify why this is being changed? I'm not seeing what it has to do with this PR.

Reply:
Per Nan's suggestion, trigger time is not set "correctly" in the current spec. But removing fake reports after source deactivation requires the trigger time to be set correctly. 

@linnan-github

Reply:
What about the current spec isn't correct? Is this something that should be fixed in a separate preliminary PR?

Reply:
Afaik, `triggerTime` in the fake report was never used currently. I am not sure a separate PR is justifiable given it's a one line change to support attribution scope. 

The trigger time in the report should be set to the window start, and reporting window to window end in the fake report. Currently it's set to window end iiuc.

Reply:
> The trigger time in the report should be set to the window start, and reporting window to window end in the fake report. Currently it's set to window end iiuc.

"Should" according to what? The trigger time doesn't appear reports.

Reply:
@linnan-github